diff --git a/node_modules/@expo/match-media/build/polyfill.native.js b/node_modules/@expo/match-media/build/polyfill.native.js
index 06d860d..0399154 100644
--- a/node_modules/@expo/match-media/build/polyfill.native.js
+++ b/node_modules/@expo/match-media/build/polyfill.native.js
@@ -1,19 +1,22 @@
 import mediaQuery from "css-mediaquery";
-import ScreenOrientation from "expo/build/ScreenOrientation/ScreenOrientation";
+import * as ScreenOrientation from "expo/build/ScreenOrientation/ScreenOrientation";
 import { Dimensions } from "react-native";
 class MediaQuery {
     constructor(query) {
         this.query = query;
         this.listeners = [];
         this.orientation = ScreenOrientation.Orientation.PORTRAIT;
+        // this.orientation = "PORTRAIT"
         this.resize = () => {
             this.updateListeners({ orientation: this.orientation });
         };
         // @ts-ignore
         (async () => {
-            const { orientation } = await ScreenOrientation.getOrientationAsync();
+            // const { orientation } = await ScreenOrientation.getOrientationAsync();
+            const { orientation } = await Promise.resolve({orientation: "PORTRAIT"});
             this.updateListeners({ orientation });
         })();
+        // this.unsubscribe = () => {}
         this.unsubscribe = ScreenOrientation.addOrientationChangeListener(({ orientationInfo: { orientation } }) => {
             this.updateListeners({ orientation });
         });
